modules/tools/aws-node-termination-handler/main.tf

```diff
@@ -0,0 +1,13 @@
+resource "helm_release" "aws-node-termination-handler" {
+  count      = var.enabled ? 1 : 0
+  repository = "https://aws.github.io/eks-charts"
+  chart      = "aws-node-termination-handler"
+  version    = "0.13.3"
+  namespace  = var.namespace
+  name       = "aws-node-termination-handler"
+  timeout    = 600
+
+  values     = [
+    file("${path.module}/values.yaml")
+  ]
+}
```

Comment:
is this our first time using `aws-node-termination-handler`?  before we were using something else like `k8s-spot-termination-handler`.  I'm sure this works fine, but was this applied to sandbox yesterday?  so we should be able to know how that worked last night.  any info on this?

Reply:
It deployed fine in sandbox and all of the logs look fine. I I don't really have a good way to test one of the spot instances being removed.

The new one also has a prometheus metrics endpoint which would be cool to look at the data there and maybe setup alerts but it's not really a priority right now.